### ISSUE TITLE
setPedAnimation() does not work when a ped is attached and floating in air

### ISSUE THREAD

**Issue by Einheit-101** [multitheftauto/mtasa-blue#463]
**Describe the bug**
Currently setPedAnimation() while being attached only works when the ped has something below his feet (i.e. he is glued on top of a car using "glue" resource).
Trying to play animations while a player is floating in the air does not work, **but setPedAnimation() will still return true.**

**To Reproduce**
1. Spawn a car
2. Walk on the car roof
3. crun attachElements(me, getPedContactElement(me), 0, 0, 3)
4. crun setPedAnimation(me, "dancing", "dnce_m_d", 1000, true, false, true, true)
**-> The animation will not play, no matter what you do.**

5. crun attachElements(me, getPedContactElement(me), 0, 0, 1)
6. crun setPedAnimation(me, "dancing", "dnce_m_d", 1000, true, false, true, true)
**-> The animation will now play, because you are in contact with the car**

**Expected behavior**
setPedAnimation() should force the animation even when the player has no contact with another object, at least if interruptable is set to false.

**MTA version 1.5.6-release-14378**

**Comment by Einheit-101** [comment]
I can no longer reproduce this with the steps to reproduce. The animation always plays even without contact element. I think this can be closed if no one else can confirm this anymore.

**Comment by qaisjp** [comment]
I vaguely recall a "recent" fix relating to attachElements (6-12 months?)

can someone track the commit down?

**Comment by Einheit-101** [comment]
Maybe
https://github.com/multitheftauto/mtasa-blue/commit/3ab471ccb3de31757741da22a60b6132461d362b

https://github.com/multitheftauto/mtasa-blue/commit/14e5d11d5a2cb980b9f3346ce069a2db2e818638

**Comment by qaisjp** [comment]
yes 14e5d11d5a2cb980b9f3346ce069a2db2e818638 is what i was thinking of. this commit is small enough for you to revert, build & test (that the bug returns). are you (or someone) able to do that?

**Comment by Einheit-101** [comment]
I cant, i still dont know how to build MTA and even then, i would not know what to do with the code 😂👌🏼

**Comment by qaisjp** [comment]
If you're interested in building for the first time, pop over to Discord and I'm sure someone will help you get started :) 

I can then tell you how to change the code to test it! You will need to have https://desktop.github.com/ installed and I think follow https://wiki.multitheftauto.com/wiki/Compiling_MTASA

**Comment by Einheit-101** [comment]
I think i will try it at some point :D
Maybe i can fix some famous bugs at some point...
